docs/reference/builder.md

```diff
@@ -1452,6 +1479,32 @@ Line 3 does not cause a cache miss because the value of `CONT_IMG_VER` is a
 constant (`hello`). As a result, the environment variables and values used on
 the `RUN` (line 4) doesn't change between builds.
 
+To demonstrate the behaviour of the predefined `ARG` statements, consider the
```

Comment:
Isn't this the same content as above? Why not have the content one place and then provide a link at the other place?